tests/e2e/logger/test_logger.py

```diff
@@ -36,3 +48,47 @@ def test_basic_lambda_logs_visible(basic_handler_fn, basic_handler_fn_arn):
     assert len(logs.get_cold_start_log()) == 1
     assert len(logs.get_log(key=custom_key)) == 2
     assert logs.have_keys(*LOGGER_LAMBDA_CONTEXT_KEYS) is True
+
+
+# test on combination of utc,tz,datefmt params
+@pytest.mark.xdist_group(name="logger")
+@pytest.mark.parametrize("utc", [False, True])
+@pytest.mark.parametrize("tz", ["US/Eastern", "UTC", "Asia/Shanghai"])
+@pytest.mark.parametrize("datefmt", ["%z", None])
+def test_lambda_tz(tz_handler_fn, tz_handler_fn_arn, utc, tz, datefmt):
+    # GIVEN
```

Comment:
add details on GIVEN WHEN THEN. This helps self-document tests. Specially useful for parametrized tests that are testing more than one scenario